clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/description/manifest/KubernetesPatchManifestDescription.java

```diff
@@ -36,7 +37,7 @@
   // This will only be a portion of a full manifest so calls to some required fields can fail.
   // Using the KubernetesManifest type makes it simpler to reuse the ArtifactReplacement logic.
   // TODO: change Orca to only send a single manifest.
-  private KubernetesManifest patchBody;
+  private KubernetesPatchBody patchBody;
```

Comment:
If someone already has a pipeline with a patch stage defined, editing this stage's description will break it. I would either: keep this field the same, and add a new one for the jsonpatch object, or, go with what you have and write a pipeline migrator. I think @jtk54 can help with that (it's not too much work).

Reply:
Or, even easier, make this an `Object` or `Map`, and decide how to cast it (using an `ObjectMapper`) based on what the strategy is in the below `KubernetesPatchOptions` object.

Reply:
Good call, updated!